src/App.js

```diff
@@ -1,22 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react'
 import Breadcrumb from './Breadcrumb'
-import './App.css';
+import './App.css'
 
-const DUMMY_BREADCRUMBS = [
-  { label: 'one', onClick: (path) => console.warn('PATH:', { path }) },
-  { label: 'two', onClick: (path) => console.warn('PATH:', { path }) },
-  { label: 'three', onClick: (path) => console.warn('PATH:', { path }) },
-  { label: 'four', onClick: (path) => console.warn('PATH:', { path }) }
-]
+function App() {
+  const [activePath, setActivePath] = useState('one/two/three')
 
-const DUMMY_ACTIVE_PATH = 'one/two'
+  const onBreadcrumbClick = path => {
+    console.log('on breadcrumb click', path)
```

Comment:
let's not have logs in source

Reply:
you had console warn statements 😛

Reply:
in the dummy app code, not the breadcrumb source